megalinter-reports/IDE-config.txt

```diff
@@ -0,0 +1,31 @@
+MegaLinter can help you to define the same linter configuration locally
```

Comment:
question: I think those files shouldn't be included, Am I right? 

I guess they were added by megalinter because `megalinter-reports` is not in `.gitignore`